clang/lib/Driver/ToolChains/Darwin.cpp

```diff
@@ -2577,6 +2577,27 @@ void AppleMachO::AddClangSystemIncludeArgs(
   }
 }
 
+void DarwinClang::AddClangSystemIncludeArgs(
+    const llvm::opt::ArgList &DriverArgs,
+    llvm::opt::ArgStringList &CC1Args) const {
+  AppleMachO::AddClangSystemIncludeArgs(DriverArgs, CC1Args);
+
+  if (DriverArgs.hasArg(options::OPT_nostdinc) ||
```

Comment:
This is a change in behavior. Previoiusly `-Xclang -nostdsysteminc` would additionally remove the framework search paths. However, I wanted to make the default framework include paths be consistent with the default header include paths. I don't believe there's a practical use case to keep the default header search paths but remove the default framework search paths, which is what `-Xclang -nostdsysteminc` would do. I think the previous code used `HeaderSearchOptions.UseStandardSystemIncludes` because that's all you get at that level of the code.